starboard/linux/x64x11/test_filters.py

```diff
@@ -19,6 +19,7 @@
 _FILTERED_TESTS = {
     # TODO(b/321315344): Re-enable once the coverage build is fixed.
     'nplb': ['PosixMemoryMapTest.CanChangeMemoryProtection',],
+    'extension_test': ['ExtensionTest.CrashHandler'],
```

Comment:
I would prefer that we not disable this test case for all x64x11 platforms.

If we do move forward with a test like the one currently added in this PR, I think we could restrict this filter to just MODULAR builds. Another idea, which I actually prefer, is to revert the changes to `starboard/shared/starboard/crash_handler.cc` from https://github.com/youtube/cobalt/pull/1330/files and instead make a change so that MODULAR builds don't even implement the CrashHandler Starboard Extension.

Reply:
I think it would be better to use the test in the filter, but for `_MODULAR_BUILD`, similar to how it is done in `cobalt/starboard/linux/shared/test_filters.py`. If we revert this commit `https://github.com/youtube/cobalt/pull/1330/files`, when `SetString` is called and it uses `InsertCrashpadAnnotation`, we would need to call the function `InstallCrashpadHandler(const std::string& ca_certificates_path)` before usage, otherwise, an error will occur:
```
  if (!SignalHandler::Get()) {
    DLOG(ERROR) << "Crashpad isn't enabled";
    return false;
  }
```